agent/websockets/shim.go

```diff
@@ -165,7 +169,7 @@ const (
           throw new Error('WebSocket is not yet opened');
         }
         this.pendingMessages.push({'id': this._sessionID, 'msg': data});
-        self.push();
```

Comment:
Minor nit: This seems like a bug-fix for the websocket issue.  If so, it might be better to submit this separately.  That way, if we have to revert the batching code due to some issue we don't accidentally revert this fix as well

Reply:
You are right, I should have done this separately. I will be more diligent about that in the future.